### DISCUSSION TITLE
Why do lookup tables take various individual bits as input instead of a bus?

### DISCUSSION THREAD

**Discussion by gbizzotto** [hneemann/Digital#589] [Q&A] [answered]
This forces the use of a splitter, and the input ports' position even incentivizes some creative gymnastics.
![Screen Shot 2020-12-23 at 09 37 54](https://user-images.githubusercontent.com/5064376/102996927-b5f64980-4502-11eb-923c-2d5698571ed0.png)

**Comment by hneemann** [comment, marked as answer]
If the input is a bus, such a component would be called a ROM. :-)
Just use a ROM instead and connect the sel input to high.

**Reply by gbizzotto** [reply]
Ah. Right. Sorry about that.

**Reply by hneemann** [reply]
No reason to apologize.
